### PR TITLE
Trade vol

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/398

### CODE REVIEW

File: octobot_trading/constants.py [hunk 1]
```diff
@@ -110,6 +110,13 @@
                               "hitbtc", "kraken", "poloniex", "bitso", "ndax", "upbit",
                               "wavesexchange"]
 
+# from https://www.coingecko.com/en/categories/stablecoins
+USD_LIKE_COINS = [
+    "USDT", "USDC", "TUSD", "BUSD", "DAI", "USD",
+    "FRAX", "USDD", "USDP", "GUSD", "LUSD", "FDUSD",
+    "ALUSD", "USDTC", "MIM", "DOLA", "BOB", "MIMATIC"
+]
```
Comment (Herklos) [inline octobot_trading/constants.py:118]: I think we should add it into constants to make it available for every other modules

Comment (GuillaumeDSM) [reply octobot_trading/constants.py:118]: It is already in constants or you mean something else ?

Comment (Herklos) [reply octobot_trading/constants.py:118]: Sorry I mean in commons

Comment (GuillaumeDSM) [reply octobot_trading/constants.py:118]: good idea, i'll move it there

**Review by Herklos — APPROVED** [review]

**Pull request merged**